_source/_data/authors.yml

```diff
@@ -854,3 +854,11 @@ arek-nawo:
   web: https://areknawo.com/
   twitter: https://twitter.com/areknawo
   github: https://github.com/areknawo
+
+mike-nachbaur:
+  full_name: Michael A. Nachbaur
+  display_name: Mike Nachbaur 
+  avatar: avatar-mike-nachbaur.jpg
+  web: https://nachbaur.com/
+  twitter: https://twitter.com/nachoman
+  github: https://github.com/nachoman
```

Comment:
you could also add a `bio: one or few lines` if you like

Reply:
Oh cool, I copied one of the other entries, and didn't notice the bio option. I'll add that in the morning.